src/libs/Navigation/AppNavigator/createCustomBottomTabNavigator/TopBar.tsx

```diff
@@ -84,4 +86,7 @@ export default withOnyx<TopBarProps, TopBarOnyxProps>({
     policy: {
         key: ({activeWorkspaceID}) => `${ONYXKEYS.COLLECTION.POLICY}${activeWorkspaceID}`,
     },
+    session: {
+        key: ONYXKEYS.SESSION,
+    },
```

Comment:
Please add a `selector` here, as you are only interested in the `authTokenType` property

Reply:
Done